sorry about forgetting that - TravisCI is complaining about a failing unrelated test though, I don't quite understand why

```
=================================== FAILURES ===================================
_ TestCommonIOCapabilities.test_write_fspath_all[to_hdf-writer_kwargs3-tables] _
...
pandas/tests/io/test_common.py:172: AssertionError
======= 1 failed, 10171 passed, 345 skipped, 6 xfailed in 424.99 seconds =======
```

--- a/doc/source/whatsnew/v0.20.2.txt
+++ b/doc/source/whatsnew/v0.20.2.txt
@@ -79,6 +79,7 @@ Reshaping
 ^^^^^^^^^
 
 - Bug in ``DataFrame.stack`` with unsorted levels in MultiIndex columns (:issue:`16323`)
+- Bug in ``pd.wide_to_long()`` where no error was raised when ``i`` was not a unique identifier (:issue:`16382`)
 
 
 Numeric

